# NOTE flask-socketio 5.0.0 and later is giving a strange error message
# check back for resolution https://github.com/miguelgrinberg/Flask-SocketIO/issues/1432
# NOTE sqlalchemy 1.4.0 and later is giving a strange error
# check back for resolution

from setuptools import setup

setup()